src/Explorer/Explorer.tsx

```diff
@@ -654,6 +690,13 @@ export default class Explorer {
     if (!notebookContentItem || !notebookContentItem.path) {
       throw new Error(`Invalid notebookContentItem: ${notebookContentItem}`);
     }
+    if (
+      notebookContentItem.type === NotebookContentItemType.Notebook &&
+      userContext.features.notebooksTemporarilyDown === false &&
+      userContext.features.phoenix === true
```

Comment:
why not use Notebookutil.isPhoenixEnabled() wherever you have this? believe thats the goal of having this util method?

Reply:
updated.